crates/symbolicator-service/src/services/sourcemap_lookup.rs

```diff
@@ -615,6 +621,7 @@ struct ArtifactFetcher {
     scope: Scope,
     source: Arc<SentrySourceConfig>,
 
+    // the two lookup indices
```

Comment:
Is this intentionally not a doc comment?

Reply:
yes, as it is rather about grouping related things together than about documenting what they individually are.